### PR TITLE
allow for classNames to be passed in

### LINKED ISSUE #112: passing classNames to tooltip-on-element doesn't work anymore
Just upgraded to 2.4.0 and now passing `classNames` into `tooltip-on-element` doesn't add that class to the div.

```hbs
{{#tooltip-on-element
   side="right"
   tooltipIsVisible=true
  event="none"
  classNames="got-it"
}}
   Some text
{{/tooltip-on-element}}
```

### PR DESCRIPTION
**Pull request by SirZach**
closes #112

### CODE REVIEW

File: tests/integration/components/tooltip-on-element-test.js [hunk 1]
```diff
@@ -40,3 +40,18 @@ test('it has the proper aria-describedby tag', function(assert) {
   assert.equal(this.$(`#${describedBy}`).text().trim(), 'Some info in a tooltip.');
   assert.equal(describedBy.indexOf('#'), '-1');
 });
+
+test('it renders passed in classes', function(assert) {
```
Comment (a15n) [inline tests/integration/components/tooltip-on-element-test.js]: this is the exact type of test that we need, but can you add it to the `pass-through-test.js` file instead? You can just add classNames to the existing test and add more `.hasClass` calls

**Comment by a15n** [comment]
This fixes the issue with `classNames`. thank you SirZach. While we're at it can you do a bit of research and see if there are any other attributes that need to be passed through? I think attributeBindings would be another that we should add coverage for during the same PR.

**Comment by SirZach** [comment]
@a15n you cannot pass in `attributeBindings`. The component itself needs to have the `attributeBindings` setup. I can take a crack with a separate PR to add all the attributes

**Comment by sir-dunxalot** [comment]
@SirZach Can you merge in master and update the test suite? Then we should be good to merge and release this improvement!

**Comment by a15n** [comment]
thanks for finding this @SirZach. We've actually addressed the issue with this PR. 

https://github.com/sir-dunxalot/ember-tooltips/pull/133

**Comment by SirZach** [comment]
@a15n sorry for my absence and thanks for following it through.

**Pull request closed without merging**